web/client/product/pages/Importer.jsx

```diff
@@ -0,0 +1,84 @@
+const PropTypes = require('prop-types');
+/**
+ * Copyright 2016, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+const React = require('react');
+const {connect} = require('react-redux');
+
+const url = require('url');
+const urlQuery = url.parse(window.location.href, true).query;
+
+const ConfigUtils = require('../../utils/ConfigUtils');
+
+const {resetControls} = require('../../actions/controls');
+
+const HolyGrail = require('../../containers/HolyGrail');
+/**
+  * @name Importer
+  * @memberof pages
+  * @class
+  * @classdesc
+  * Importer allow a user with admin permissions to import data in GeoServer.
+  * This page have to be configured in appConfig `pages`.
+  * The app is available at http://localhos:8081/#/importer
+  *
+  * @example
+  * // localConfig configuration example
+  * "plugins": {
+  *  "importer": [
+  *         // ...
+  *         {
+  *             "name": "Importer",
+  *            "cfg": {} // see plugin configuration
+  *         }
+  *     ]
+  * }
+*/
+
+class ImporterPage extends React.Component {
```

Comment:
Maybe it's time to have some basic page to reuse, I think most of the pages we have are now identical, except some configuration option. What do you think?

Reply:
There are some differences that can not be put at common during this activity. I would prefer to do a task like this using different, dedicated time. . 

The only ones that look really the same are Importer and RulesManager (with anyway some exclusion that is not trivial). The other have some actions, lifecycle, actions that are not easy to put together.

Reply:
Please, open a separated issue for this

Reply:
did it. here it is
https://github.com/geosolutions-it/MapStore2/issues/3841